firefox-ios/Client/Frontend/TrackingProtection/CertificatesViewController.swift

```diff
@@ -0,0 +1,276 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Common
+import Shared
+import SiteImageView
+
+import Security
+import CryptoKit
+import X509
+import SwiftASN1
+
+struct CertificateKeys {
+    static let commonName = "CN"
+    static let country = "C"
+    static let organization = "O"
+}
+
+struct CertificatesUX {
```

Comment:
I think we'd like to embed the `CertificateKeys` and `CertificatesUX` inside the `CertificatesViewController`. I double checked in slack, and that sounds like the style people prefer to move toward for UX constants (example: the generalized `UX` struct in `CreditCardBottomSheetViewController`).

Reply:
Ok, I can do that but that will cause some errors in my cell because I use also there some UX values. May I separate them? UX for cell and UX for Controller?

Reply:
Yes, I think that makes sense, since none of the constants are used by both the cell and the controller (as far as I can tell).

So you can add a UX struct inside your cell (with just the constants needed in the cell), and a UX struct inside the view controller (with just the constants needed in the view controller). 👍